site/content/userguide/usage/branches-and-ci.md

```diff
@@ -17,25 +17,28 @@ for the application, environment, _and tag_. Leaving out the tag for the master
 The app-tag feature provides the following facilities:
 
 * GC is scoped to a tag such that it doesn't try to delete mainline objects or objects created with other tags.
-* It exposes the tag value as a jsonnet external variable (`qbec.io/tag`) so your components can use this to create per-tag objects with unique 
+* It exposes the tag value as a jsonnet external variable (`qbec.io/tag`) so your components can use this to create per-tag objects with unique
   names.
 * It exposes the name of the default namespace used for the run (`qbec.io/defaultNs`) and, on request, can automatically
   suffix this value for you.
-  
+
 ## Rules of engagement
 
 To be able to use this feature:
 
 * You can only create namespaced objects. The only cluster scoped object you can create is a namespace itself.
   This feature is not meant to be used with cluster scoped objects.
-  
+
 * You must either modify the name of _every_ object you create based on the supplied tag name _or_
-  create a new namespace for every tag and create the objects in it. Failure to do so will cause non-deterministic and 
+  create a new namespace for every tag and create the objects in it. Failure to do so will cause non-deterministic and
   unexpected GC behavior.
 
 * The tag passed to qbec cannot have special characters like `/`. It must conform to what can be used as a label
   value in Kubernetes. qbec will validate this and will not let you proceed otherwise.
 
+* `qbec` is built with safety in mind. All command with side-effects require user confirmation. This behaviour can be overridden by using
```

Comment:
this is probably not the correct place for this material since it pertains to branch builds specifically.